ui/app/src/components/ToolsPanelListItemButton/ToolsPanelListItemButton.tsx

```diff
@@ -43,14 +44,15 @@ export default function ToolsPanelListItemButton(props: ToolsPanelListItemButton
     secondaryActionIcon = <ChevronRounded />,
     onSecondaryActionClick
   } = props;
+  const { formatMessage } = useIntl();
   return (
     <ListItem button onClick={onClick} ContainerComponent="div">
       <ListItemIcon>
-        <Icon fontSize="small" className={icon?.baseClass} />
+        <SystemIcon icon={icon} />
       </ListItemIcon>
-      <ListItemText primary={title} secondary={subtitle} />
+      <ListItemText primary={formatMessage({ id: title, defaultMessage: title })} secondary={subtitle} />
```

Comment:
Please test if the built app shows this without throwing exceptions. Need to test in default locale (en) and any other too (e.g. es) to make sure.